test/e2e/common/host_path.go

```diff
@@ -45,7 +45,7 @@ var _ = Describe("[sig-storage] HostPath", func() {
 	   Testname: Host path, volume mode default
 	   Description: Create a Pod with host volume mounted. The volume mounted MUST be a directory with permissions mode -rwxrwxrwx and that is has the sticky bit (mode flag t) set.
 	*/
-	framework.ConformanceIt("should give a volume the correct mode [NodeConformance]", func() {
+	framework.ConformanceIt("should give a volume the correct mode [NodeConformance][Privileged]", func() {
```

Comment:
we have removed the `privileged := true` below... do we still need the new label `[Privileged]` here?

Reply:
@dims yes, you are right. I will remove the `[Privileged]` label